modules/admin-server/src/main/kotlin/com/sphereon/oid/fed/server/admin/config/MonitoringConfig.kt

```diff
@@ -34,29 +35,41 @@ class MonitoringConfig {
         val severity =
             if (memoryUsagePercent >= memoryWarningThresholdPercent) Severity.Warn else Severity.Info
 
+        val dbMetrics = PlatformSqlDriver.Companion.getConnectionMetrics()
+
         val message = buildString {
             append("System Health: ")
             append("Uptime=${uptime.toHours()}h${uptime.toMinutesPart()}m, ")
             append("Memory=${usedMemory / 1024 / 1024}MB/${totalMemory / 1024 / 1024}MB($memoryUsagePercent%), ")
             append("Processors=${runtime.availableProcessors()}, ")
             append("Heap=${memoryMBean.heapMemoryUsage.used / 1024 / 1024}MB, ")
-            append("Threads=${threadMBean.threadCount}(Peak:${threadMBean.peakThreadCount}, Daemon:${threadMBean.daemonThreadCount})")
+            append("Threads=${threadMBean.threadCount}(Peak:${threadMBean.peakThreadCount}, Daemon:${threadMBean.daemonThreadCount}), ")
+            append("DB Connections: Active=${dbMetrics["Active Connections"] ?: "N/A"}, ")
+            append("Idle=${dbMetrics["Idle Connections"] ?: "N/A"}, ")
+            append("Total=${dbMetrics["Total Connections"] ?: "N/A"}")
+        }
+
+        val contextMap = mutableMapOf(
```

Comment:
I thought this could be immutable, but you want to reflect the system uptime on the logs